BUILD.gn

```diff
@@ -5,23 +5,31 @@
 # This target will be built if no target is specified when invoking ninja.
 group("default") {
   testonly = true
-  deps = [
-    "//flutter",
-  ]
+  if (target_os == "wasm") {
+    deps = [
+      "//third_party/skia/modules/canvaskit:canvaskit"
+    ]
+  } else {
+    deps = [
+      "//flutter",
+    ]
+  }
```

Comment:
nit: Can we do this in Flutter's BUILD.gn instead? 

It's fine to do it here but I think it will be easier to iterate on this in general if more of the rules exist in the flutter repo.